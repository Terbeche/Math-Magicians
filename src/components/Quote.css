.quote-container {
  margin: 7rem 3rem;
  font-size: 2rem;
  text-align: center;
  color: saddlebrown;
}
