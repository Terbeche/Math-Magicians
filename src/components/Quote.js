/* eslint-disable jsx-a11y/label-has-associated-control */
import { React } from 'react';
import './Quote.css';

const Quote = () => (
  <div className="quote-container">
    <p>
      Mathematics is not about numbers, equations, computations,
      <br />
      or algorithms: it is about understanding. -William Paul Thurston
    </p>
  </div>
);

export default Quote;
