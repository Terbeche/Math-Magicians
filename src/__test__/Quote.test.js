import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../components/Quote';

describe('Testing the Quote', () => {
  it('Home is rendered', () => {
    const quote = renderer
      .create(<BrowserRouter><Quote /></BrowserRouter>)
      .toJSON();
    expect(quote).toMatchSnapshot();
  });
});
