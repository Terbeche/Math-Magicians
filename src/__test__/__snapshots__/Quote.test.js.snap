// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Quote Home is rendered 1`] = `
<div
  className="quote-container"
>
  <p>
    Mathematics is not about numbers, equations, computations,
    <br />
    or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
